int64 time
int64 devnum
float32[] setpoints
float32[] measured
float32[] input_pressure